hpack/table.py

```diff
@@ -171,17 +171,13 @@ def search(self, name, value):
             - ``(index, name, value)`` for perfect matches.
         """
         partial = None
-
-        header_name_search_result = HeaderTable.STATIC_TABLE_MAPPING.get(name)
-        if header_name_search_result:
-            index = header_name_search_result[1].get(value)
-            if index is not None:
-                return index, name, value
-            else:
-                partial = (header_name_search_result[0], name, None)
-
-        offset = HeaderTable.STATIC_TABLE_LENGTH + 1
-        for (i, (n, v)) in enumerate(self.dynamic_entries):
+        for (i, (n, v)) in enumerate(HeaderTable.STATIC_TABLE):
+            if n == name:
+                if v == value:
+                    return i + 1, n, v
+                elif partial is None:
+                    partial = (i + 1, n, None)
```

Comment:
I think this is from where I mucked up a rebase from 119. I'm probably going to just restart with master since the change is so minor.